src/keyszer/config_api.py

```diff
@@ -41,6 +41,21 @@
 # multipurpose timeout
 _TIMEOUTS = TIMEOUT_DEFAULTS
 
+# delay globals to fix Unicode entry sequence and macro failures
+THROTTLE_DELAYS = {'unicode_delay_ms': 0, 'keystroke_delay_ms': 0}
+
+def throttle_delays(unicode_delay_ms=0, keystroke_delay_ms=0):
+    _ud, _kd = unicode_delay_ms, keystroke_delay_ms
+    if _ud != 0 or _kd != 0: debug(f'Throttle delays: {unicode_delay_ms = }, {keystroke_delay_ms = }', ctx="II")
+    if _ud <= 100 and _ud >= 0 and isinstance(_ud, int): THROTTLE_DELAYS.update({'unicode_delay_ms': _ud})
+    else: debug(f'ERROR: Unicode delay throttle must be int 0 to 100 ms. Defaulting to 0 ms.', ctx="EE")
+    if _kd <= 150 and _kd >= 0 and isinstance(_kd, int): THROTTLE_DELAYS.update({'keystroke_delay_ms': _kd})
+    else: debug(f'ERROR: Keystroke delay throttle must be int 0 to 150 ms. Defaulting to 0 ms.', ctx="EE")
+
+# for use with throttle delays
+def sleep_ms(usec):
+    return time.sleep(usec / 1000)
```

Comment:
I think the input here needs to be `ms`... `usec` is really a micro-second, not a millisecond.

Reply:
Corrected. I picked that up mistakenly from initially copying `usleep`. Which means even the name of `usleep` is incorrect, since it's only dividing by 1000 too. 

I even corrected someone recently that `ms` is milliseconds, not microseconds. 🤦🏽‍♂️